data/mods/MindOverMatter/professions.json

```diff
@@ -60,6 +60,34 @@
       "female": [ "bra", "panties" ]
     }
   },
+  {
+    "type": "profession",
+    "id": "psi_photokinetic_magician",
+    "name": "Magician",
+    "description": "Ever since you were a young kid, you just loved magic.  Growing up you managed to become a professional magician and even performed at a few big shows.  You were practicing a new trick when the Cataclysm hit, when you realized you don't need to practice anymore.  You can just do it, for real.",
```

Comment:
Stage magicians should have some social skill 🤔

Reply:
https://github.com/CleverRaven/Cataclysm-DDA/pull/69092/commits/868479fe679aa87778d031fc05ad17e61f74e1a0